src/components/generic/design/paginator.svelte

```diff
@@ -18,12 +18,6 @@
 			upperBoundary--;
 		}
 
-		console.log(lowerBoundary, upperBoundary);
-
-		if (upperBoundary === lowerBoundary) {
```

Comment:
Leave this if statement in. The paginator will break with low result counts if this check isn't in there.

Reply:
@lvdlee It currently skips 2 if the pagecount is 3.

I tested the change with 1-2-3-4 results and they all looked fine. Bigger results were fine aswell @

Reply:
@Tunnelbliick so it's completely tested and working like this if I understand correctly?

Reply:
@lvdlee i tested it a bunch yesterday, but ill double down and test it again once im home.

Reply:
![image](https://user-images.githubusercontent.com/38663241/181594834-b9141ba7-315f-437c-a5f0-69597f3213ac.png)
![image](https://user-images.githubusercontent.com/38663241/181595511-70ba9383-01de-42d5-8ce7-97175904800a.png)
![image](https://user-images.githubusercontent.com/38663241/181595560-4cbac89c-c569-4e4a-b311-75106b48f92a.png)

Reply:
@lvdlee high result counts move as normal aswell so i dont think this is an issue.